### PR TITLE
Add Systems Engineering Management Plan Draft

### PR DESCRIPTION
**Pull request by amsmith-pro**

### CODE REVIEW

File: docs/semp.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+
+## :busts_in_silhouette: Code Collaboration
+
+### :muscle: Contributing
+
+The process of contributing to a GitHub repository is well documented online.
+1. Make a copy of the repository on the local machine
+2. Initialize a "branch" using git
+    - A branch is a set of changes with a label, e.g. `readme-updates`
+3. Edit the relevant document(s)
```
Comment (GoodluckH) [inline docs/semp.md]: Might be worth noting that the branch name should also have the github username (e.g. `goodluckh/readme-updates`). This makes it easier for searching and filtering.

File: docs/semp.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+
+Document containing:
+- State Machine Sequence Diagrams
+- Explanation of libraries
+- Explanation of global variables
+- Explanation of memory allocations
+
+Relevant:
+- [Arrow Markdown SDD Template](./templates/sdd.md)
+
```
Comment (GoodluckH) [inline docs/semp.md]: Broken link

File: docs/semp.md [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+#### :mailbox_with_mail: Interface Control Document (ICD)
+
+This document details:
+- Inputs and Outputs of a Module
+- Packet Formats
+- Frameworks and Libraries
+
+Relevant:
+- [Arrow Markdown ICD Template](./templates/icd.md)
+
```
Comment (GoodluckH) [inline docs/semp.md]: Broken link

Let's make a PR to include these templates first, then we will come back to this pr for approval.

**Review by GoodluckH — CHANGES_REQUESTED** [review]

File: docs/semp.md (hunk 2 above)
Comment (amsmith-pro) [reply docs/semp.md]: Changed to the svc-template-rust/docs/sdd.md file

File: docs/semp.md (hunk 3 above)
Comment (amsmith-pro) [reply docs/semp.md]: Added a 🚧, still determining the ICD format

File: docs/semp.md (hunk 1 above)
Comment (amsmith-pro) [reply docs/semp.md]: Updated

File: docs/semp.md (hunk 2 above)
Comment (amsmith-pro) [reply docs/semp.md]: https://github.com/Arrow-air/se-services/pull/1/commits/158ed78fed2da6cd5c4e468c9f21d75e25cb421d

File: docs/semp.md (hunk 3 above)
Comment (amsmith-pro) [reply docs/semp.md]: https://github.com/Arrow-air/se-services/pull/1/commits/158ed78fed2da6cd5c4e468c9f21d75e25cb421d

File: docs/semp.md (hunk 1 above)
Comment (amsmith-pro) [reply docs/semp.md]: https://github.com/Arrow-air/se-services/pull/1/commits/f1ce881223e9986116a69d481dd93e4c408d444c

**Review by GoodluckH — APPROVED** [review]

**Review by owlot — APPROVED** [review]

**Pull request merged**